packages/sdk/src/generateSnapshotFromManifest.ts

```diff
@@ -49,6 +49,14 @@ export const simpleJoinRemoteEntry = (rPath: string, rName: string): string => {
 };
 
 export function inferAutoPublicPath(url: string): string {
+  if (url.includes('/_next/')) {
+    return url
```

Comment:
SDK should not be modified for framework specifics like next.js. 

Either theres a bug in how public path is calculated - it should be solved in a generic way. or this should be changed in the nextjs-mf package if its next specific.

Reply:
@ScriptedAlchemy thank you for guide! I will give it a try!